src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/FinishedOnTime/WhenMakingDuePayments.cs

```diff
@@ -328,7 +328,7 @@ public void ThenItShouldMake16To18FirstIncentivePayments()
             TestDataHelper.AddProvider(ukprn);
```

Comment:
unused IT.Entities using statement

Reply:
the same ukprn and commitmentId is used throughout the class. Consider refactoring to single constant fields.

Reply:
consider the same with some of the other variables used within tests (e.g. startDate)